### DISCUSSION TITLE
TFT Display

### DISCUSSION THREAD

**Discussion by codeput** [rern/rAudio#791] [General]
Hello! 
What must be the minimum resolution of the TFT display to display all the information?

In my case I don't see the menus below the cover art.

Thank you.
![IMG_20220202_110822](https://user-images.githubusercontent.com/17404321/152133944-7f812702-74c4-4ff1-9071-4d87595a4fc9.jpg)

**Comment by EarsofTin** [comment]
Perhaps Settings (gear icon)>Playback>"Bars always on" will work for you.  Makes menu bars appear on my phone.

**Reply by codeput** [reply]
Thank You for reply me, It doesn't work on the tft screen, due to the geometry of the page, however I fixed it by scaling the chromium image from the settings.
